Civi/Financeextras/Setup/Manage/AccountsReceivablePaymentMethod.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Civi\Financeextras\Setup\Manage;
+
+class AccountsReceivablePaymentMethod extends AbstractManager {
+
```

Comment:
Label/Title/Name  `accounts_receivable` and ` Accounts Receivable` are repeated a lot in the method. Would it be better if we create constant variables for these values and reuse these values in other classes too?